e2e/cypress/integration/enterprise/cloud/onboarding/next_steps_tips_spec.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @onboarding @cloud_only
+
+describe('Cloud Onboarding - Next Steps Tips', () => {
+    let townSquarePage;
+    let sysadmin;
+
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            townSquarePage = `/${team.name}/channels/town-square`;
+        });
+
+        cy.apiAdminLogin().then((res) => {
+            sysadmin = res.user;
+        });
+    });
+
+    beforeEach(() => {
+        // # Login as sysadmin and set all steps to false
+        const preference = {
+            user_id: sysadmin.id,
+            category: 'recommended_next_steps',
+            value: 'false',
+        };
+        const adminSteps = [
+            'complete_profile',
+            'notification_setup',
+            'team_setup',
+            'invite_members',
+            'enter_support_email',
+            'hide',
+            'skip',
+        ];
+
+        cy.apiSaveUserPreference(adminSteps.map((step) => ({...preference, name: step})));
+        cy.visit(townSquarePage);
+
+        cy.get('.NextStepsView__button:contains(Skip Getting Started)').scrollIntoView().should('be.visible').click();
+    });
+
+    it('Show incident collaboration', () => {
```

Comment:
I've added a new test case in Zephyr for this. Please see [MM-T4147](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T4147). There are two steps -- one for admin and the other one for regular user.

```suggestion
    it('MM-T4147_1 For system admin', () => {
```

Then, please a separate test block that checks cards/contents for regular user
```javascript
    it('MM-T4147_2 For regular user', () => {
```